mkdocs.yml

```diff
@@ -126,8 +129,6 @@ nav:
                   - Overview: tre-templates/pipeline-templates/overview.md
                   - Pipeline Schema: tre-templates/pipeline-templates/pipeline-schema.md
 
-      - Deploying the AzureTRE using CI/CD: using-tre/pipelines/pipelines.md
```

Comment:
I think this needs to stay here, as isn't just for quick start.

Reply:
@LizaShak , we just moved this section technical guide. I would agree with @marrobi to keep the section as a separate sub-section under technical guide.